src/content/staking/withdrawals/index.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Staking withdrawals
+description: Page summarizing what staking push withdrawals are, how they work, and what stakers need to do to get their rewards
+lang: en
+template: staking
+image: ../../../assets/staking/leslie-withdrawal.png
+alt: Leslie the rhino with her staking rewards
+sidebarDepth: 2
+summaryPoints:
+  - The Shanghai upgrade brings staking withdrawals to Ethereum
+  - Validator operators must provide a withdrawal address to enable
+  - Rewards payments are automatically sent to the address provided every few days
+  - Fully exited validators will automatically receive their remaining balance
+---
+
+<UpgradeStatus dateKey="page-upgrades-withdrawals">
+  Staking withdrawals will be enabled through the Shanghai/Capella upgrade. This upgrade is expected to take place in the first half of 2023. <a href="#when">More below</a>
+</UpgradeStatus>
+
+The Shanghai/Capella upgrade enables **staking withdrawals** on Ethereum, allowing people to unlock rewards and optionally fully withdrawal funds from staking. Rewards payments will automatically and regularly be sent to a provided withdrawal address linked to each validator. Users can also exit staking entirely, unlocking their full validator balance.
+
+**Note to validator operators**: If you did not provide a withdrawal address on initial deposit, you will need to upgrade withdrawal credentials before any withdrawals will be enabled for your validator account. The [Staking Launchpad](https://launchpad.ethereum.org/withdrawals) has more details on when and how to do this.
+
+## Staking rewards {#staking-rewards}
+
+**Rewards payments** are automatically processed for active validator accounts with a maxed out effective balance of 32 ETH, who are accumulating rewards.
+
+Any balance above 32 ETH earned through rewards does not actually contribute to principle, or increase the sway of this validator on the network, and is thus automatically withdrawn as a rewards payment every few days. Aside from providing a withdrawal address one time, these rewards do not require any action from the validator operator, nor do they consume gas (no associated transaction fee) or take up existing block space.
+
+### How did we get here? {#how-did-we-get-here}
+
+Over the past few years Ethereum has undergone several network upgrades transitioning to a network secured by ETH itself, instead of energy-intensive mining as it once was. Participating in consensus on Ethereum is now known as "staking", as participants have voluntarily locked up ETH, placing it "at stake" for the ability to participate in the network. Following the rules rewards users, while attempts to cheat can be penalized.
```

Comment:
Any reason you're not mentioning PoS or PoW here? Seems like a good opportunity to link out to those pages...

Reply:
Simplifying language. Intentionally tried to keep this copy at a more human-centric level, and easier reading level.

I think we could still link out though, but also @konopkja has done some work and has a [content guide](https://www.notion.so/efdn/Content-guide-based-on-collected-feedback-edfff575d1754dee9482130bec46b97e) on Notion which notes that users tend to get bogged down by our overuse of links. So, I started by minimizing the number of things we link out to in the initial pass.

As we get more finalized on the copy and structure, I think it's worth taking a look at the page and seeing what makes sense to link out to in a non-overburdening or distracting way.

Reply:
Just did a quick check and we're in the range of ~1400 words... per the suggestions in the guide of about 5 links per 1000 words, we would target about 7 links (which I don't include the further reading links at the bottom). We have 6 already... I know it's just a guide, and SEO is also worth noting, but I do agree we just don't want to go too far with them (such as linking to dev docs) for a page like this.